plugins/woocommerce-admin/client/products/fills/pricing-section/pricing-field-taxes-class.tsx

```diff
@@ -82,6 +78,6 @@ export const PricingTaxesClassField = () => {
 					} ) ) }
 				/>
 			) }
-		</CollapsibleContent>
+		</>
```

Comment:
I think we could remove the use of `Fragment` here, or is this required because its slot-filled?

Reply:
Responded to this [here](https://github.com/woocommerce/woocommerce/pull/36646/files#r1089478258).